core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.ts

```diff
@@ -70,7 +72,8 @@ export class InputResponsePairComponent {
   ) {}
 
   ngOnInit(): void {
-    this.decodedProfilePicture = decodeURIComponent(this.profilePicture);
+    this.decodedProfilePicturePng = this.profilePicturePngDataUrl;
+    this.decodedProfilePictureWebp = this.profilePictureWebpDataUrl;
```

Comment:
Why is this needed, cannot you just use `profilePicturePngDataUrl` directly?

Reply:
Yep! I have updated this part.